packages/theming/src/examples/theme.md

```diff
@@ -0,0 +1,217 @@
+
+Conceptually, this means you could override Garden's `blue` hue to be a
+completely non-blue set of shades or provide random shade ordering that doesn't
+fall in line with Garden's 100 (extra light) through 800 (extra dark) structure.
+We would discourage you from mutating the palette in this manner as it
+weakens semantics and reduces the likelihood of successful component color
+styling. When you find yourself using `palette` theming to surgically tweak
+component colors (i.e. "I want to change the background color on button
+hover"), consider using **Components** theming described above.
+
+The following snippet shows how Garden's palette can be modified and extended
+with colors from Material-UI.
+
+```jsx static
+import { ThemeProvider } from '@zendeskgarden/react-theming';
+import { Button } from '@zendeskgarden/react-buttons';
+import { colors } from '@material-ui/core';
+
+const theme = {
+  colors: {
+    dangerHue: 'deepOrange'
+  },
+  palette: {
+    blue: colors.blue,
+    deepOrange: colors.deepOrange
+  }
+};
+
+<ThemeProvider theme={theme}>
+  <>
+    <Button>Using material blue</Button>
+    <Button danger>Using material deep orange</Button>
+  </>
+</ThemeProvider>;
+```
+
+### Shadows
+
+The `shadows` portion of the default theme contains three functions:
+
+- `sm` and `md`: each take an RGBA `color` parameter and are typically used
+  to apply Garden's `:focus` box shadow. Values under the `shadowWidths` theme
+  object are used to determine shadow thickness.
+- `lg`: takes `offsetY`, `blurRadius`, and `theme` parameters for applying a
+  faint kale-colored drop shadow to components that appear to float above the
+  surface (i.e. modals, notifications, etc.).
+
+### Space
+
+The final frontier of Garden's theming system, `space.base` is the essential
+factor used to compute our base-4 component system. This value gives you
+sweeping control over component padding and margins. Pixel sizes `xxs`
+through `xxl` are added for convenience. Remember, to update other theming
+values (i.e. `borderRadii` & `lineHeights`) to correspond with your `base` of choice.
+
+### `defaultTheme` object
+
+The following reference displays Garden's `defaultTheme` in its entirety.
+
+```jsx noeditor
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { defaultTheme } = require('@zendeskgarden/react-theming/src');
+
+const StyledTheme = styled.pre`
+  margin: 0;
+  padding: 0;
+  font-family: ${props => props.theme.fonts.mono};
+  font-size: ${props => props.theme.fontSizes.mono.sm};
+  overflow: hidden;
+`;
+
+const replacer = (key, value) => {
+  let retVal = value;
+
+  if (typeof value === 'function') {
+    const fn = value.toString();
+    const start = fn.indexOf('(');
+    const end = fn.indexOf('\n') - 2;
+
+    retVal = `${fn.substring(start, end)} => expression`;
+  }
+
+  return retVal;
+};
+
+<Well recessed>
+  <StyledTheme>{JSON.stringify(defaultTheme, replacer, '  ')}</StyledTheme>
```

Comment:
TIL about the stringify replace function I've also just passed `null` and never considered what it was for.